### ISSUE TITLE
Handle filling of decimal

### ISSUE THREAD

**Issue by jclark** [ballerina-platform/nballerina#967]
I don't think we handle filling when member type is decimal. This should fill to zero.

We want to avoid this causing decimal library to get linked in.

**Comment by poorna2152** [comment]
example:
```
public function main() {
    decimal[] d = [];
    d[1] = 0.1;
    io:println(d[1]);
}
```
Gives: `panic: no filler value`

**Comment by heshanpadmasiri** [comment]
closed with #1153
